tools/WinObjC.Language/WinObjC.Language.targets

```diff
@@ -6,4 +6,65 @@
   <ImportGroup>
     <Import Project="$(MsBuildThisFileDirectory)msvc\starboard-sdk.targets" Condition="Exists('$(MsBuildThisFileDirectory)msvc\starboard-sdk.targets')" />
   </ImportGroup>
+
+  <!-- Make sure that the output artifacts are included in the build. This means adding include paths and link libraries for it.
+
+     NOTE: It is important that this information is added at the end in the .targets instead of in the .props so that consumers don't
+     accidentally stomp it and get really confused. This is contrary to a configurable type property that a consumer may want / need to adjust. 
+     To depend on this package means the consumer gets these these things.  -->
+
+  <ItemGroup Condition="'$(ConfigurationType)' != 'StaticLibrary'">
+    <_ReferencesFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.winmd"/>    
+    <_ReferencesFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.winmd"/>
+
+    <_ReferencesFromPackage Include="$(MSBuildThisFileDirectory)\deps\prebuilt\$(TargetOsAndVersion)\$(PlatformTarget)\*.winmd"/>
+    <_ReferencesFromPackage Include="$(MSBuildThisFileDirectory)\deps\prebuilt\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.winmd"/>
+
+    <_LibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.dll"/>    
+    <_LibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.dll"/>
+
+    <_LibsFromPackage Include="$(MSBuildThisFileDirectory)\deps\prebuilt\$(TargetOsAndVersion)\$(PlatformTarget)\*.dll"/>
+    <_LibsFromPackage Include="$(MSBuildThisFileDirectory)\deps\prebuilt\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.dll"/>
+
+    <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.lib"/>    
```

Comment:
>_ImportLibsFromPackage [](start = 5, length = 22)

Seems like a bit of a misnomer. This will pick up vcclang.lib, among other things, which isn't an import library.